README.adoc

```diff
@@ -299,11 +298,12 @@ servers:
 
 // Later this should be changed to:
 // For more information about which elements you can filter, see the MicroProfile 1.3 API.
+// https://openliberty.io/docs/ref/microprofile/1.3/
```

Comment:
I realize users can't see this comment right now, but we'd probably want the link to be a hyperlink from "MicroProfile 1.3 API" or maybe "MicroProfile 1.3 API documentation" instead of giving the URL itself.

Maybe something like...see the MicroProfile 1.3 API https://openliberty.io/docs/ref/microprofile/1.3/[documentation].

Reply:
Should be fixed by replacing the link to the GitHub repo with a link to OL JavaDocs for MP 1.3.